lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -161,19 +195,15 @@ public static function get_fonts_dir() {
 	 *
 	 *     @type string $path    Path to the directory.
 	 *     @type string $url     URL for the directory.
-	 *     @type string $subdir  Sub-directory of the directory.
-	 *     @type string $basedir Base directory.
-	 *     @type string $baseurl Base URL.
 	 * }
 	 * @return array Modified upload directory.
 	 */
 	public static function set_upload_dir( $defaults ) {
-		$defaults['basedir'] = WP_CONTENT_DIR;
-		$defaults['baseurl'] = content_url();
-		$defaults['subdir']  = '/fonts';
-		$defaults['path']    = self::get_fonts_dir();
-		$defaults['url']     = $defaults['baseurl'] . '/fonts';
+		$fonts_dir_settings = self::get_fonts_dir_settings();
+		$fonts_sub_dir      = self::get_multi_site_fonts_sub_dir();
 
+		$defaults['path'] = rtrim( $fonts_dir_settings['dir'], '/' ) . $fonts_sub_dir;
+		$defaults['url']  = untrailingslashit( $fonts_dir_settings['url'] ) . $fonts_sub_dir;
```

Comment:
We're assuming that `$fonts_dir_settings['dir']` and `$fonts_dir_settings['url']` also exist/are OK here as well - should we check them before setting these values?